notebooks/official/workbench/spark/spark_sample_notebook.ipynb

```diff
@@ -0,0 +1,1219 @@
+{
```

Comment:
<p>The Spark DataFrame created during this project will be stored in BigQuery. The data will be written first to the <s>bucket in </s>Google Cloud Storage(GCS) <strong>bucket</strong> and then it is loaded it to BigQuery. &nbsp;A GCS bucket must be configured to indicate the temporary data location. </p><p>Set the name of your Cloud Storage bucket below. It must be unique across all Cloud Storage buckets.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>